Sources/_Datadog_Private/ObjcAppLaunchHandler.m

```diff
@@ -4,60 +4,94 @@
 * Copyright 2019-2020 Datadog, Inc.
 */
 
-#import <UIKit/UIKit.h>
 #import "ObjcAppLaunchHandler.h"
+#import <sys/sysctl.h>
+#import <UIKit/UIKit.h>
+
+// `AppLaunchHandler` aims to track some times as part of the sequence described in Apple's "About the App Launch Sequence"
+// https://developer.apple.com/documentation/uikit/app_and_environment/responding_to_the_launch_of_your_app/about_the_app_launch_sequence
 
-@interface AppLaunchTimer : NSObject
-@property NSDate *frameworkLoadTime;
-// Knowing how this value is collected and that it is written only once, the use of
-// `atomic` property is enough for thread safety. It guarantees that neither partial
-// nor garbage value will be returned.
-@property (atomic) NSTimeInterval timeToApplicationBecomeActive;
```

Comment:
I believe `atomic` is already the default for a `@property`. That being said, I don't think this was necessary due the inherent sequencing of setting/getting this property throughout the App lifecycle.

Reply:
So it looks like `LaunchTimeProviderTests.testThreadSafety()` disagrees with my assumptions 🤔